.github/has-parameters-tree-changes.sh

```diff
@@ -0,0 +1,100 @@
+#! /usr/bin/env bash
+
+RED='\033[0;31m'
+BLUE='\033[0;34m'
+COLOR_RESET='\033[0m'
+
+
+# openfisca-france expected paths for parameters
+EXPECTED_PATHS=(
+    "openfisca_france"
+    "openfisca_france/parameters"    
+    "openfisca_france/parameters/chomage"
+    "openfisca_france/parameters/chomage/allocation_retour_emploi"
+    "openfisca_france/parameters/chomage/allocations_assurance_chomage"
+    "openfisca_france/parameters/chomage/allocations_chomage_solidarite"
+    "openfisca_france/parameters/chomage/preretraites"
+    "openfisca_france/parameters/geopolitique"  # premier niveau bloqué seulement ; n'existe pas dans les barèmes IPP
```

Comment:
Existe désormais [côté barèmes IPP](https://framagit.org/french-tax-and-benefit-tables/baremes-ipp-yaml/-/tree/master/parameters)